src/storage/MonitoringStore.ts

```diff
@@ -36,7 +38,7 @@ export class MonitoringStore<T extends ResourceStore = ResourceStore>
 
     // Both the container contents and the resource itself have changed
     try {
-      const container = getParentContainer(identifier);
+      const container = this.containerManager.getParentContainer(identifier);
```

Comment:
Can we get away with the `try`/`catch` now if we do `if not isParentContainer`?

Reply:
True, I didn't look in-depth enough into what was happening there.